Formula/libui.rb

```diff
@@ -0,0 +1,50 @@
+class Libui < Formula
+  desc "portable GUI library for C"
+  homepage "https://github.com/andlabs/libui"
+  url "https://github.com/andlabs/libui/archive/alpha3.1.tar.gz"
```

Comment:
There *was* `homebrew-head-only`, but we not longer provide an official tap for non-stable formulae. Until this gets a stable release, the best action is probably to keep it in a personal tap.